diffs/platformref.diff

```diff
@@ -0,0 +1,490 @@
+html/en/beats/libbeat/master/beats-reference.html
```

Comment:
diff contains some issues I did not address because they were in old versions of the relnotes. will communicate requirement to dev team for having space before link